coding-concepts.js

```diff
@@ -7,28 +7,28 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer 5
+// b) Verify and explain: .push adds a value to the end of the array and returns index
 
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log the index of 10
+// b) Verify and explain: .length counted the index of the string starting from 0
```

Comment:
.length counts the number of characters in a string or elements in an array. Because strings and arrays are zero-indexed, the last index is actually 9.